webapp/app.ironcalc.com/frontend/src/App.tsx

```diff
@@ -21,6 +23,8 @@ import { IronCalc, IronCalcIcon, Model, init } from "@ironcalc/workbook";
 
 function App() {
   const [model, setModel] = useState<Model | null>(null);
+  const [showWelcomeDialog, setShowWelcomeDialog] = useState(false);
+  const [isTemplateOnlyDialog, setIsTemplateOnlyDialog] = useState(false);
```

Comment:
This seems a bit like a Hack to reuse the WelcomeDialog as a TemplateDialog. I think it would be clearer to have two different dialogs WelcomeDialog and TemplateDialog. Maybe one opens the other. Maybe you can base the code of both in some common dialog